src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -101,14 +101,14 @@ def load_arguments(self, _):
         c.argument('source', options_list=['--source', '-s'], arg_type=get_enum_type(['file', 'appconfig', 'appservice']), validator=validate_import, help="The source of importing. Note that importing feature flags from appservice is not supported.")
         c.argument('yes', help="Do not prompt for preview.")
         c.argument('skip_features', help="Import only key values and exclude all feature flags. By default, all feature flags will be imported from file or appconfig. Not applicable for appservice.", arg_type=get_three_state_flag())
-        c.argument('content_type', help='Content type of all imported items.')
+        c.argument('content_type', help='Content type of all imported items. If you are using JSON content type, make sure the values are in valid JSON format.')
```

Comment:
> If you are using JSON content type, make sure the values are in valid JSON format. [](start = 76, length = 83)

nit: We will throw error for invalid json type, no need to remind here. Considering if we support yaml or other types, we can't remind them all in argument help message.